crates/holochain_state/src/event.rs

```diff
@@ -0,0 +1,80 @@
+use crate::prelude::*;
```

Comment:
This whole module can be removed at any point if you like. This is the part we don't agree on being useful.

Reply:
If the usefulness is doubted, I'd tend to exclude this module from the PR.

Reply:
I personally think it's quite useful, but if everybody else does not, somebody should make a commit to delete it.